src/js/components/molecules/cta-block/cta-block.ts

```diff
@@ -0,0 +1,25 @@
+import { Vue, Component, Prop } from 'vue-property-decorator';
+import { CtaBlock as ICtaBlock } from '@ultimaker/ultimaker.com-model-definitions/dist/molecules/cta/CtaBlock';
+import { Button } from '@ultimaker/ultimaker.com-model-definitions/dist/atoms/cta/Button';
+import { ContentLink } from '@ultimaker/ultimaker.com-model-definitions/dist/atoms/cta/ContentLink';
+
+@Component({
+    name: 'cta-block',
+    template: require('./cta-block.html'),
+})
+
+export default class CtaBlock extends Vue implements ICtaBlock {
+    @Prop({ type: Array, required: true }) ctas!: ICtaBlock['ctas'];
+
+    get isButton ()  {
+        return this.ctas.filter(elem => {
+            return elem instanceof Button;
+        })
+    }
+
+    get isContentLink ()  {
+        return this.ctas.filter(elem => {
+            return elem instanceof ContentLink;
+        })
+    }
+}
```

Comment:
Its better to name these buttons and contentLinks so its more clear it is a collection of items.